assets/css/edd-admin.css

```diff
@@ -2206,3 +2256,127 @@ td.edd_order_price {
 .edd-paypal-account-status .dashicons-no {
 	color: #d63638;
 }
+
+/* Notifications
+-------------------------------------------------------------- */
+.edd-overlay {
+	position: fixed;
+	z-index: 1052;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 160px;
+	background-color: #141b38;
+	opacity: .5;
+	transition: .5s;
+}
+
+.edd-slide-in {
+	transform: translateX(100%) !important;
+	-webkit-transform: translateX(100%) !important;
+}
+
+#edd-notifications-panel {
+	background-color: white;
+	height: 100%;
+	width: 100%;
+	max-width: 570px;
+	position: fixed;
+	z-index: 1053;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	overflow-x: hidden;
+	transition: .5s;
+
+	transform: translateX(0%);
+	-webkit-transform: translateX(0%);
+}
+body.admin-bar #edd-notifications-panel {
+	top: 32px;
+}
+@media screen and (max-width: 600px) {
+	body.admin-bar #edd-notifications-panel {
+		top: 46px;
+	}
+}
+
+#edd-notifications-header {
+	display: flex;
+	align-items: center;
+	padding: 0 30px;
+	color: #fff;
+	background-color: #0c5d95;
+}
+#edd-notifications-header h3 {
+	color: #fff;
+	flex: 1;
+}
+#edd-notifications-header .edd-close {
+	background: none;
+	border: none;
+	color: #fff;
+	cursor: pointer;
+}
+
+#edd-notifications-body {
+	padding: 30px;
+}
+
+.edd-notification {
+	display: flex;
+	gap: 20px;
+	margin-bottom: 20px;
+}
+.edd-notification--icon {
+	color: #00aa63;
+}
+.edd-notification--icon.edd-notification--icon-info {
+	color: #005ae0;
+}
+.edd-notification--icon.edd-notification--icon-warning {
+	color: #f18200;
+}
+.edd-notification--icon.edd-notification--icon-error {
+	color: #df2a4a;
+}
+.edd-notification--body {
+	flex: 1;
+}
+.edd-notification--header {
+	align-items: center;
+	display: flex;
+	justify-content: space-between;
+	gap: 5px;
+	margin-bottom: 7px;
+}
+.edd-notification--title {
+	color: #141b38;
+	flex: 1;
+	font-size: 16px;
+	font-weight: 600;
+	margin: 0;
+}
+.edd-notification--date {
+	color: #8c8f9a;
```

Comment:
This color could be darker (contrast ratio is 3.22; 4.5 is preferred). Same color as for the `--dismiss` button.

Reply:
Updated to `#71747e` in https://github.com/awesomemotive/easy-digital-downloads/pull/8941/commits/c7b6f0090e8c0037d709cc5adf36c332251aaf52 -- how's that?